src/guides/v2.3/config-guide/deployment/pipeline/technical-details.md

```diff
@@ -136,9 +149,14 @@ On your production system:
 1. If you installed components that changed the database schema, run `magento setup:upgrade --keep-generated` to update the database schema and data, preserving generated static files.
 1. To set system-specific settings, use either the `magento config:set` command or environment variables.
 1. To set sensitive settings, use either the `magento config:sensitive:set` command or environment variables.
+1. Use the `magento setup:di:compile` command to compile code.
+1. Use the `magento setup:static-content:deploy` command to update static file view files.
```

Comment:
These two steps already happened on the build server, so they shouldn't be re-run again here, right? The idea of this way of working is that this runs fast with as little downtime as possible, adding these two command here will defeat this purpose.

Reply:
Updated